ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/EthProtocolManager.java

```diff
@@ -379,6 +381,7 @@ private void handleStatusMessage(final EthPeer peer, final MessageData data) {
       } else if (mergePeerFilter.isPresent()) {
         final boolean disconnected = mergePeerFilter.get().disconnectIfPoW(status, peer);
         if (disconnected) {
+          LOG.info("here we are disconnecting at request from mergePeerFilter disconnectIfPoW");
```

Comment:
similar thing about the 'here we are' bit and peer info

Reply:
done